Formula/v2ray.rb

```diff
@@ -0,0 +1,49 @@
+class V2ray < Formula
+  desc "Platform for building proxies to bypass network restrictions"
+  homepage "https://v2fly.org/"
+  url "https://github.com/v2fly/v2ray-core/archive/v4.32.1.tar.gz"
+  sha256 "f5553ffd04ff226573e8132d9beaa63f4d8f4882eba047225b33620848bc6917"
+  license "MIT"
+  head "https://github.com/v2fly/v2ray-core.git"
+
+  depends_on "go" => :build
+
+  resource "config" do
+    url "https://github.com/v2fly/v2ray-core/raw/v4.32.1/release/config/config.json"
+    sha256 "e4498e48725cb8e835aa384b98e494579b1f04059576fdaf43ea0579498b7edd"
+  end
+
+  resource "geoip" do
+    url "https://github.com/v2fly/geoip/releases/download/202011150541/geoip.dat"
+    sha256 "11b7c3bfc5715c42d26b0e4bcf51d38c157eae9ab4b9e8391d702681e385dbcd"
+  end
```

Comment:
I doubt this is legal. Could you check if this doesn't violate the MaxMind license agreement?

Reply:
It's CC-BY-SA-4.0: https://dev.maxmind.com/geoip/geoip2/geolite2/#License

Reply:
In that case it needs to be added to the license statement, and the `v2fly/geoip` repo can't use just MIT so they probably need to be notified of that.

Reply:
v2fly/geoip#4